.automation/generated/linter-versions.json

```diff
@@ -72,7 +72,7 @@
     "powershell_formatter": "7.4.2",
     "prettier": "3.3.2",
     "proselint": "0.14.0",
-    "protolint": "0.50.2",
+    "protolint": "0.49.8",
```

Comment:
Protolint has a renovate update pending, please wait before merging this (that is a downgrade).